src/GoogleFitOnFhir/Clients/GoogleFit/GoogleFitDataImporterContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace GoogleFitOnFhir.Clients.GoogleFit
+{
+    public class GoogleFitDataImporterContext
+    {
+        /// <summary>
+        /// Sets the results limit of a Dataset request
+        /// </summary>
+        public static int GoogleFitDatasetRequestLimit => 10;
```

Comment:
Can we add the ParallelTaskOptions.MaxConcurrency value here too?

Reply:
Sure thing.